ChromeExtension/groupsReview.html

```diff
@@ -32,48 +32,42 @@
 
 <body id="body">
     <div style="padding: 20px">
-        <div class="group-div">
-            <div class="text-left">
-                <p class="newlink-group newlink-select-review">Put new links in this group</p>
-                <select id="listOfGroups" class="form-control listOfGroups-review">
-                    <option id="loadingOption" value="Loading">Loading...</option>
-                </select>
-            </div>
-
-            <center>
-                <p style="margin-bottom: 5px;" id="result">&nbsp;</p>
-            </center>
-
-            <div id="dialog" title="Error">
-                <div id="networkError"></div>
-            </div>
-        </div>
+        <center>
```

Comment:
While we are in here, mind replacing these HTML4 `<center>` tags? Eg, with `<div style="text-align: center;">`

Reply:
done

Reply:
Thanks!